packages/amplify-auth-sdk/package.json

```diff
@@ -25,22 +25,23 @@
     "jws": "^3.2.2",
     "open": "^6.1.0",
     "pluralize": "^7.0.0",
+    "semver": "^6.0.0",
     "snooplogg": "^2.0.1",
     "source-map-support": "^0.5.12",
     "uuid": "^3.3.2"
   },
   "devDependencies": {
     "appcd-gulp": "^2.1.0",
+    "keytar": "^4.4.1",
```

Comment:
Minor, but pin this too and keep in lock with the `keytar` prop?

Reply:
Fixed!